### ISSUE TITLE
GQA Accuracy

### ISSUE THREAD

**Issue by HellwayXue** [ronilp/mac-network-pytorch-gqa#5]
Could you please tell me about the accuracy of the model under the GQA task？ I only reached 45%

**Comment by chengstark** [comment]
Hello, I'm running it on gqa as well, I ran 12 epochs reached about 40 percent acc. How many epochs did you run?

**Comment by simpleshinobu** [comment]
same question about accuracy. On the GQA leaderboard, the accuracy is > 50%

**Comment by simpleshinobu** [comment]
> Hello, I'm running it on gqa as well, I ran 12 epochs reached about 40 percent acc. How many epochs did you run?

Hello, I find the strange.
Just mute one line in model_gqa.py:
self.embed.weight.requires_grad = False
Though I do not know why the default is freezing the word embedding.

**Comment by jeasinema** [comment]
> > Hello, I'm running it on gqa as well, I ran 12 epochs reached about 40 percent acc. How many epochs did you run?
> 
> Hello, I find the strange.
> Just mute one line in model_gqa.py:
> self.embed.weight.requires_grad = False
> Though I do not know why the default is freezing the word embedding.

Thanks for the catch! I'm just asking: what is your reproduced result on GQA with this fix?
